### PR TITLE
feat(google-maps): Marker Customization Options

### PR DESCRIPTION
**Pull request by theproducer**
This PR adds support for more Google Maps marker customization options:
- Default marker tint color (native only)
- Customer marker icons on web and native (including anchor, sizing and origin)

### CODE REVIEW

File: google-maps/e2e-tests/android/app/build.gradle [hunk 1]
```diff
@@ -1,21 +1,5 @@
-plugins {
```
Comment (ItsChaceD) [inline google-maps/e2e-tests/android/app/build.gradle:1]: Was this supposed to be removed? Without it the Android app will throw a manifest error.

File: google-maps/e2e-tests/android/app/build.gradle [hunk 2]
```diff
@@ -1,21 +1,5 @@
...
-    id 'com.google.android.libraries.mapsplatform.secrets-gradle-plugin'
-}
-
 apply plugin: 'com.android.application'
 
-def getApiKey() {
```
Comment (ItsChaceD) [inline google-maps/e2e-tests/android/app/build.gradle:7]: This is also a part of the secrets gradle plugin

File: google-maps/e2e-tests/android/app/build.gradle [hunk 3]
```diff
@@ -30,7 +14,6 @@ android {
              // Default: https://android.googlesource.com/platform/frameworks/base/+/282e181b58cf72b6ca770dc7ca5f91f135444502/tools/aapt/AaptAssets.cpp#61
             ignoreAssetsPattern '!.svn:!.git:!.ds_store:!*.scc:.*:!CVS:!thumbs.db:!picasa.ini:!*~'
         }
-        manifestPlaceholders = [MAPS_API_KEY:"${getApiKey()}"]
```
Comment (ItsChaceD) [inline google-maps/e2e-tests/android/app/build.gradle:33]: Also a part of secrets plugin

File: google-maps/e2e-tests/android/build.gradle [hunk 4]
```diff
@@ -7,9 +7,9 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.1.1'
-        classpath 'com.google.gms:google-services:4.3.5'
-        classpath "com.google.android.libraries.mapsplatform.secrets-gradle-plugin:secrets-gradle-plugin:2.0.0"
```
Comment (ItsChaceD) [inline google-maps/e2e-tests/android/build.gradle:12]: Need this for setting `MAPS_API_KEY`

File: google-maps/e2e-tests/src/pages/Markers/MarkerCustomizations.tsx [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+        zoom: 7,
+      });
+      setMarkerIds(ids);
+      setCommandOutput(`${ids.length} color markers added`);
+    } catch (err: any) {
+      setCommandOutput(err.message);
+    }
+  };
+
+  const removeAllMarkers = async () => {
```
Comment (ItsChaceD) [inline google-maps/e2e-tests/src/pages/Markers/MarkerCustomizations.tsx:303]: This only removes the last markers added, not all on the map. Not sure if it's necessary to fix, but would help being able to test all functions at once

**Review by ItsChaceD — CHANGES_REQUESTED** [review]

**Review by IT-MikeS — APPROVED** [review]

**Review by ItsChaceD — APPROVED** [review]

**Pull request merged**